src/__tests__/parser.ts

```diff
@@ -1076,13 +1070,11 @@ describe('parser', () => {
       });
 
       it('should allow filtering by parent declarations', () => {
-        const propFilter: PropFilter = prop => {
+        const propFilter: PropFilter = (prop) => {
           if (prop.declarations !== undefined && prop.declarations.length > 0) {
-            const hasPropAdditionalDescription = prop.declarations.find(
-              declaration => {
-                return !declaration.fileName.includes('@types/react');
-              }
-            );
+            const hasPropAdditionalDescription = prop.declarations.find((declaration) => {
+              return !declaration.fileName.includes('@types/react');
+            });
```

Comment:
```suggestion
            const hasPropAdditionalDescription = prop.declarations.find((declaration) => !declaration.fileName.includes('@types/react'));
```

Reply:
I've added a `.prettierrc` to the project to match the settings in package.json & updated the files